src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="index.html"
+      method="post"
+      class="big__bar"
+      data-qa="big"
+      >
```

Comment:
```suggestion
    >
```

Reply:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/91217030/ad692d5a-4311-4329-835e-6b730e1c47f3)
he said me doing like this